registration/include/pcl/registration/sample_consensus_prerejective.h

```diff
@@ -256,7 +272,7 @@ namespace pcl
         * \param sample_indices the resulting sample indices
         */
       void 
-      selectSamples (const PointCloudSource &cloud, int nr_samples, std::vector<int> &sample_indices);
+      selectSamples (const PointCloudSource &cloud, int nr_samples, std::vector<int> &sample_indices) const;
```

Comment:
`nr_samples` can be unsigned here, also `sample_indices`, although maybe `index_t` is even better for that? There are more `int` indices in here, but maybe it'd be more appropriate to address thta in a future PR.

Reply:
I'll open another PR to change them to unsigned after this PR is merged.